keras_nlp/layers/multi_segment_packer.py

```diff
@@ -187,22 +217,40 @@ def _combine_inputs(self, segments):
         dtype = segments[0].dtype
         batch_size = segments[0].nrows()
         start_value = tf.convert_to_tensor(self.start_value, dtype=dtype)
+        sep_value = tf.convert_to_tensor(self.sep_value, dtype=dtype)
         end_value = tf.convert_to_tensor(self.end_value, dtype=dtype)
 
-        start_column = tf.fill((batch_size, 1), start_value)
-        end_column = tf.fill((batch_size, 1), end_value)
-        ones_column = tf.ones_like(start_column, dtype=tf.int32)
+        start_values_tensor = tf.repeat(
```

Comment:
these names are a little confusing `start_value` is already a tensor. should we co back to `_column` naming?

`start_column`, `end_column`, `sep_column`?